static/styles/settings.css

```diff
@@ -337,14 +337,6 @@ td .button {
         margin-top: 9px;
     }
 
-    .loading_indicator_spinner {
-        width: 30%;
-        height: 30px;
-        margin-top: 7px;
-        vertical-align: middle;
-        display: inline-block;
-    }
```

Comment:
Is this CSS class only used for this element?  I'd want to be sure of that before deleting its definition.

Reply:
So, this CSS is applied to *all* loading spinners in settings overlay, which isn't desirable in `confirm_dialog`'s spinner. I am going to add it again since it's used by other elements.
Maybe I should change the name of the class to apply this to specific elements, but not sure if that's the right way. Any suggestions?